google/resource_compute_instance_group_manager.go

```diff
@@ -33,10 +33,40 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 
 			"instance_template": &schema.Schema{
 				Type:             schema.TypeString,
-				Required:         true,
+				Optional:         true,
 				DiffSuppressFunc: compareSelfLinkRelativePaths,
 			},
 
+			"version": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"instance_template": &schema.Schema{
+							Type:             schema.TypeString,
+							Required:         true,
+							DiffSuppressFunc: compareSelfLinkRelativePaths,
+						},
+
+						"target_size_fixed": &schema.Schema{
```

Comment:
I was not able to set `ConflictsWith` because it is a property of an element inside a list.
I saw a sample when the list is at the root level:
`ConflictsWith: []string{"rolling_update_policy.0.max_surge_percent"}`

But I don't see how I can make it work for target_size (which is a list of 1 element) inside version (which is a list too).
I tried (naively):
- `ConflictsWith: []string{"percent"}` and `ConflictsWith: []string{"fixed"}`
- `ConflictsWith: []string{"target_size.0.percent"}` and `ConflictsWith: []string{"target_size.0.fixed"}`

My feeling is that I need to find the right indice to edit the correct `version` with something like that:
`ConflictsWith: []string{"version.<X>.target_size.0.percent"}`. Is that what I need to do ?

Reply:
Oh, you're right- that isn't possible at the moment.